2_process/src/process_fcn.R

```diff
@@ -3,14 +3,12 @@
 #' The `process()` function filters the data and assigns colors/pch values based on
 #' model_type.
 #' 
-#' @param input_data chr, file path where the raw data is saved
-#' 
-#' @param output_file chr, file path of the subfolder where processed data will be saved 
+#' @param input_data chr, input object (raw data)
```

Comment:
Instead of `input_data`, I think calling this argument `input_file` might be better. That way it is very clear that it isn't the data itself, but rather the filepath to the data.

Reply:
Totally! Thanks!